styles/Header.module.css

```diff
@@ -0,0 +1,65 @@
+.header {
```

Comment:
The menu should be aligned center.
Prototype:
<img width="341" alt="image" src="https://user-images.githubusercontent.com/70834897/219576590-21dd6592-f4f2-4362-a50b-92bfca6b0090.png">
this pr:
<img width="215" alt="image" src="https://user-images.githubusercontent.com/70834897/219576783-28b4c24e-f2c0-41e1-acdb-45b8b1ef31be.png">

Reply:
I mean vertical direction